recaf-launcher/build.gradle

```diff
@@ -0,0 +1,52 @@
+buildscript {
+    repositories {
+        jcenter()
+    }
+    dependencies {
+        classpath 'net.sf.proguard:proguard-gradle:6.3.0beta1'
+    }
+}
+
+dependencies {
+    implementation common_io
+    implementation joptsimple
+    implementation directories
+    implementation gson
+    implementation logback_classic
+}
+
+jar {
+    manifest {
+        attributes('Main-Class': 'dev.xdark.recaf.Launcher')
+    }
+}
+
+task fatJar(type: Jar) {
+    with jar
+    archiveBaseName.set(archiveBaseName.get() + '-fat')
+    from {
+        configurations.runtimeClasspath.collect { zipTree(it) }
+    }
+    manifest {
+        attributes('Main-Class': 'dev.xdark.recaf.Launcher')
+    }
+}
+
+task shrinkJar(type: proguard.gradle.ProGuardTask) {
+    dontwarn() // Should we really enable that?
+    libraryjars "${System.getProperty("java.home")}/lib/rt.jar"
```

Comment:
While we will be running this on Java 8 for now, what would it look like if we wanted to run on Java 11?